modules/microshift-kubeconfig-overview.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * microshift/microshift_configuring/microshift-kubeconfig.adoc
+
+:_content-type: CONCEPT
+[id="kubeconfig-files_{context}"]
```

Comment:
The file name should match this title more closely

Reply:
Going to go with "= Kubeconfig files for configuring cluster access" since we have a prohibition on "overview" in titles.